plugin-server/src/cdp/consumers/cdp-cyclotron-worker.test.ts

```diff
@@ -203,5 +200,74 @@ describe('CdpCyclotronWorker', () => {
             const results = await processor['loadHogFunctions']([createExampleInvocation(fn2, globals)])
             expect(results).toEqual([])
         })
+
+        describe('thread relief', () => {
```

Comment:
This is the main big safety test that now 100% confirms that the exec doesn't block the thread